source/parameters/referencechange.md

```diff
@@ -0,0 +1,48 @@
+%---------------------------------------------------------------------------------------------------
+(s:ref.change.params)=
+## ReferenceChangeP: Change in Reference Parameters
```

Comment:
Parameter groups that explicitly refer to "reference" orbit values will stimulate a lot of discussion.  There has been a stated goal of separating the lattice description from the dynamics description, and here these are explicitly mixed.

Reply:
Yes but for `normalized` parameters like `Kn1`, the reference energy is needed to convert to a field.